packages/schemas/alterations/next-1698728945-user-logto-data.ts

```diff
@@ -0,0 +1,20 @@
+import { sql } from 'slonik';
+
+import type { AlterationScript } from '../lib/types/alteration.js';
+
+const alteration: AlterationScript = {
+  up: async (pool) => {
+    await pool.query(sql`
+      alter table users
+        add column if not exists logto_data jsonb not null default '{}'::jsonb;
+    `);
```

Comment:
should we also migrate the existing mfa skip data in the custom_data field?

Reply:
this feature is not released yet, so no need to alter.